internal/server/server.go

```diff
@@ -450,6 +480,10 @@ func (s *Server) Refresh(ctx context.Context) error {
 		return errors.Wrap(err, "unable to sync sources")
 	}
 
+	if dr, ok := s.rep.(repo.DirectRepository); ok && s.maint != nil {
+		s.maint.refresh(ctx, dr, false)
+	}
```

Comment:
When `ok` is true, `s.maint != nil` is always true as well, so isn't the interface check for `s.rep.(repo.DirectRepository)` is redundant ?

Reply:
done, simplified the code even more